scmail.py

```diff
@@ -119,23 +121,34 @@ def register(ctx):
     else:
         print('Registration fail.\nError is: {}'.format(res['error']))
```

Comment:
Note: Dictionary accesses like this throw an error if the key doesn't exist. While this is _probably_ safe, it might be better to use the dictionary's `.get(...)` method to ensure that this doesn't bring the whole server down.

```python
print('Registration fail.\nError is: {}'.format(res.get("error", ""))
```

Reply:
Thanks, I will try this method.